### PR TITLE
Links: Add link to eos-icons.

### PR DESCRIPTION
**Pull request by Abhirup-99**

### CODE REVIEW

File: dist/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "EOS",
   "version": "1.0.0",
   "description": "EOS plugins.",
-  "summary": "EOS plugins.",
+  "summary": "Thank you for using EOS Icons. For any issues, feel free to check out https://eos-icons.com/.",
   "languages": ["en"],
   "author": "Abhirup Pal",
   "helpUrl": "abhiruppalmethodist@gmail.com",
```
Comment (cyntss) [inline dist/manifest.json:9]: please change the `helpUrl` to the repo URL 
https://github.com/EOS-uiux-Solutions/eos-icons-adobe-xd

File: dist/manifest.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "name": "EOS",
   "version": "1.0.0",
   "description": "EOS plugins.",
```
Comment (cyntss) [inline dist/manifest.json:5]: lets change this one to `EOS icons XD plugin`

File: dist/manifest.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "name": "EOS",
```
Comment (cyntss) [inline dist/manifest.json:3]: let's change this to `EOS-icons`

**Comment by Abhirup-99** [comment]
@cyntss I have updated all the fields.

**Review by cyntss — APPROVED** [review]

**Pull request merged**